compiler/src/iree/compiler/Codegen/LLVMGPU/Utils/LLVMGPUUtils.cpp

```diff
@@ -543,7 +543,8 @@ getContractionLayoutImpl(IREE::GPU::MMAScheduleAttr schedule,
   // logical warp+thread using the subgroup/thread basis, so the subgroup
   // basis should remain the same for all A/B/C matrix.
 
-  auto [intrinsicM, intrinsicN, intrinsicK] = mmaAttr.getMNKShape();
+  auto [intrinsicM, intrinsicN, intrinsicK] =
```

Comment:
I'm very suspicious of this

You should be `dyn_cast`ing for `MMAInterfaceAttr` and calling that `getMNKShape()`, and then separately handling `ScalingMMAAttr`, and then `return failure();`if it's neither of those